### DISCUSSION TITLE
Problems uploading a Scorm to Chamilo

### DISCUSSION THREAD

**Discussion by AngelRaGu** [chamilo/chamilo-lms#5740] [General]
Hello, I hope you can help me, I have version 1.11.26 of Chamilo installed and it does not allow me to upload a Scorm file, it loads the entire file and tells me that the upload was successful, but immediately a screen appears with error 500. I have tried several ways to correct it but I have not been able to do so.
![error 500](https://github.com/user-attachments/assets/c8870327-c3f9-46eb-ab61-477cd90b56e0)
![Error](https://github.com/user-attachments/assets/67f335a6-eb90-4fcb-9ebf-b0926af50c5b)

**Comment by NicoDucou** [comment]
Hi @AngelRaGu 

There is a problem in Chamilo with the names contening special caracters.
So the short solution is renaming your scorm package to avoid the accents and the spaces and any other special caracter.
The correction is done in Chamilo in the git version and will be included in the next Chamilo release 1.11.28
So if you have the competencies you can try to update from git or apply the specific correction searching for the commit that correspond.
Otherwise change your scorm name and wait for 1.11.28 to be released.

Best regards

**Comment by Oscaruix** [comment]
I've tried the proposed solution but I'm still getting this error.

**Request**
GET /main/upload/upload.php?cidReq=DEMOSTRATIVA&id_session=0&gidReq=0&gradebook=0&origin=&from=bigUpload&name=autoestima.zip&use_max_score=1 HTTP/1.1
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7
Accept-Encoding: gzip, deflate, br, zstd
Accept-Language: en-US,en;q=0.9
Connection: keep-alive
Cookie: agenda_cookies={%22view%22:%22month%22%2C%22start%22:%222025-06-01%22}; ch_sid=uasdu99ka9e0jorhss83f1cmfu
Host: campus.getahead.mx
Referer: https://campus.getahead.mx/main/upload/index.php?cidReq=DEMOSTRATIVA&id_session=0&gidReq=0&gradebook=0&origin=&curdirpath=/&tool=learnpath
Sec-Fetch-Dest: document
Sec-Fetch-Mode: navigate
Sec-Fetch-Site: same-origin
Sec-Fetch-User: ?1
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/137.0.0.0 Safari/537.36
sec-ch-ua: "Google Chrome";v="137", "Chromium";v="137", "Not/A)Brand";v="24"
sec-ch-ua-mobile: ?0
sec-ch-ua-platform: "Windows"

**Response**
HTTP/1.1 500 Internal Server Error
Date: Fri, 06 Jun 2025 01:18:57 GMT
Server: Apache
**X-Powered-By: PHP/8.2.28**
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate
Pragma: no-cache
X-Content-Type-Options: nosniff
Content-Length: 0
Connection: close
Content-Type: text/html; charset=UTF-8



xhr.onreadystatechange @ https://campus.getahead.mx/main/inc/lib/javascript/bigupload/js/bigUpload.js:256
sendFileData @ https://campus.getahead.mx/main/inc/lib/javascript/bigupload/js/bigUpload.js:263
xhr.onreadystatechange @ https://campus.getahead.mx/main/inc/lib/javascript/bigupload/js/bigUpload.js:211

In the console I get this error:
resource: the server responded with a status of 500 ()

**could it be related to the PHP version been used?**

**Comment by Coursenligne** [comment]
Please note that:
- By default you need to upload SCORM 1.2; There are settings to have Chamilo SCORM2004 complient, but I don't have it by heart
- Chamilo 1.11 is designed inder PHP 7.4. Yous then need 7.4.33 PHP version. I read here you are using 8.2. You may occur some issues.
I hope this helps.
